### ISSUE TITLE
Dead clicks on side bar - Desktop web app - Portrait Monitors

### ISSUE THREAD

**Issue by DeckardHoliday** [bluesky-social/social-app#1219]
**Describe the bug**

In portrait mode the sidebar collapses. When opened, clicking on the following items result in no actions:

- Profile Image
- Display Name
- User Handle
- Follower Count
- Follows Count
- 'Profile' navigation button

Portrait desktop in use is 1080x1920

**To Reproduce**

1. Open up bsky.app on Portrait monitor
2. Open the collapsed side bar
3. Click on the items laid out in bug description to observe lack of behavior

**Expected behavior**

1. Clicking on the profile image, display name, user handle, or "Profile" navbar item should navigate to the user's profile (bsky.app/porfile/[user]
2. Clicking on the 'Followers' item should navigate to the user's followers list (bsky.app/profile/[user]/followers)
3. Clicking on the "Follows" item should navigate to the user's follows list (bsky.app/profile/[user]/follows)

**Screenshots**

[Dead click areas boxed in red](https://drive.google.com/file/d/1M-HDMGCEDs-IlBSqf61yEaGVpi11s0hB/view?usp=sharing)

**Details**

 - Platform: Desktop Windows, Chromium browsers (Edge and Chrome tested), Firefox, Assuming all browsers are affected
 - Platform versions:
```
Firefox: V116.0.2 64-bit
Edge: Version 115.0.1901.203 (Official build) (64-bit)
Chrome: Version 115.0.5790.173 (Official Build) (64-bit)

App version: Bsky Build V 1.45.0
```

**Comment by pfrazee** [comment]
Yeah we basically don't have responsive styling in yet

**Comment by ansh** [comment]
This should be fixed by #1437
